states/load.js

```diff
@@ -51,7 +51,14 @@ module.exports = {
     const sfxRoot        = `${audioRoot}/sfx`;
 
     this.load.audio('main-soundtrack',  `${soundtrackRoot}/title-or-intro.wav`);
-    this.load.audio('scene-soundtrack', `${soundtrackRoot}/intro-strings.wav`);
+
+    // String snippets used in the cutscenes
+    for (let x = 0; x < 9; x++) {
+      this.load.audio(
+        `string-snippet-${x}`,
```

Comment:
Can we indicate that this is a sound in some way? Also, IMO, it's better to name the asset after its use, rather than what it is (makes it easier to swap out later), so in this case `scene-soundtrack-snippet-${x}`, or something?